tests/flow/tests_onnx.py

```diff
@@ -567,7 +567,7 @@ def test_synchronization(self):
 
         def launch_redis_and_run_onnx(con, proc_id, pipes):
             my_pipe = pipes[proc_id]
-            port = 6380 + proc_id  # Let every subprocess run on a fresh port.
+            port = 6380 + 30*proc_id  # Let every subprocess run on a fresh port (safe distance for RLTEST parallelism).
```

Comment:
didn't you remove RLTest parallelism?

Reply:
Yes... But I don't think that this addition is bad, as we might wan't to use the parallelism in the future. WDYT?